app/apps-v2.json

```diff
@@ -46,5 +46,15 @@
       "type": "addon",
       "url": "https://github.com/mikehenrty/fxos-shb-plus",
       "revision": "9b4ed3b83735a3fcd764b317499efde5b1e64f68"
+    },
+    "https://shinglyu.github.io/fxos-back-button/manifest.webapp": {
+      "manifestURL": "https://shinglyu.github.io/fxos-back-button/update.webapp",
```

Comment:
The `package_path` inside of your update.manifest still points to the old `package.zip`. Please make it point to the `new-package.zip` instead.